starknet-ff/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
```

Comment:
Same here we don't need this

Reply:
file removed as this was added during rebase/merge and now not needed.